plugin/mlsdbprovider.cpp

```diff
@@ -286,6 +287,52 @@ void MlsdbProvider::timerEvent(QTimerEvent *event)
 
 void MlsdbProvider::calculatePositionAndEmitLocation()
 {
+    tryFetchOnlinePosition();
```

Comment:
This feature needs to be selectable by the user.  We need to add a new location setting key (different from the current cell_id_positioning one) which allows the user to enable or disable the online MLS lookup.  Both for privacy reasons and battery/data use reasons.